### ISSUE TITLE
No trigger on first function argument when using double quotes (gopls)

### ISSUE THREAD

**Issue by nordicatlantic** [ray-x/lsp_signature.nvim#217]
Hi,

I have a pretty generic setup with `nvim-cmp`, `lsp_signature` has been installed and configured without any custom configuration. I just invoke it with require/setup and that's it.

For reference, I got Neovim latest nightly dev (0.8.0+), nvim-cmp, nvim-lspconfig and nvim-autopairs along with this plugin.

Please watch the video: the problem is shown in the second part of this video.

https://user-images.githubusercontent.com/107055811/189869691-5de2af60-8d4f-4709-83a8-f4f6067cac5b.mov

**Comment by ray-x** [comment]
I did some tests, it seems to be a bug with gopls. As the signature does not return a valid signature until cursor moved to `"addr"`

**Comment by nordicatlantic** [comment]
Yeah, interesting. Seems to be working fine with e.g. `sumneko_lua`.
But flickering popup window persists in both cases (when moving between chars in Insert mode).

**Comment by ray-x** [comment]
As for flickering. I feel it was caused by your setup.
Do you have a minium init.lua I can reproduce?

**Comment by nordicatlantic** [comment]
Hi,

Sorry for the delay, I will reproduce something minimal a bit later (as currently my setup is pretty decoupled) or upload it to GitHub as well. Meanwhile, I noticed some patterns that might be helpful to debug this issue. The idea is that when you go to INSERT mode having a cursor positioned within some parameter, there's no flickering but also the floating window doesn't move **above or below**. On the other hand, if you happen to enter INSERT mode in some other place or temporarily leave the arguments section of a function, it can move in all four directions, except it starts to flicker.

https://user-images.githubusercontent.com/107055811/191003057-e8473a63-4bd4-4631-b0b9-7c3e5ffcce28.mov
